.semaphore/npm.yml

```diff
@@ -15,8 +15,9 @@ blocks:
         commands:
           - checkout
           - nvm use
-          - yarn install
+          - bash ~/workos-node/bin/restore-or-install
       jobs:
         - name: yarn publish
           commands:
+            - echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
             - yarn publish --access=public
```

Comment:
Who's token is in there?

Reply:
Yours?

Reply:
Lets see what happens, but I think we need to create an account without 2fa, or without 2fa for publishing enabled.

Pretty sure npm will request the OTP from the stdin while the job runs.